tests/test_common/test_cxtgeo_lowlevel.py

```diff
@@ -80,3 +80,70 @@ def test_well_mask_shoulder(depth, logseries, distance, expected):
     print(mask)
     assert res == 0
     assert (mask == expected).all()
+
+
+@pytest.mark.parametrize(
+    "xcoords, ycoords, zcoords, xval, yval, method, zexpected",
+    [
+        (
+            [0.0, 1.0, 0.0, 1.0],
+            [0.0, 0.0, 1.0, 1.0],
+            [1.0, 2.0, 3.0, 4.0],
+            0.5,
+            0.5,
+            2,  # bilinear, nonrotated
```

Comment:
Define these numbers as module constants in some module?

Reply:
In some other file (module)? Not practical with tests?

Reply:
If you have the magic numbers defined in a module for usage in that module, you could import the constants from the module into this test code (but then the test code would not catch unintentional changes to the constants..)

Reply:
I'll keep this as is